src/scripts/burgerMenu.ts

```diff
@@ -0,0 +1,61 @@
+const Menu = () => {
+  const menu = document.querySelector(".menu") as HTMLDivElement;
+  const menuBar1 = document.querySelector(".menu_bar1") as HTMLDivElement;
+  const menuBar2 = document.querySelector(".menu_bar2") as HTMLDivElement;
+  const menuBar3 = document.querySelector(".menu_bar3") as HTMLDivElement;
+  const sidebar = document.querySelector(".sidebar") as HTMLDivElement;
+  const sidebarel = document.querySelectorAll(
+    ".sidebarel"
+  ) as NodeListOf<HTMLDivElement>;
+
+  const closeSidebar = () => {
```

Comment:
Can we move these to a dedicated `.css` file?

Reply:
added updates as per the review, PTAL.